bin/build-metadata.js

```diff
@@ -331,24 +331,20 @@ function insertCountryAliases( data ) {
 }
 
 /**
- * Converts a given object to a JS string, then compiles and executes the given string and compares the two with deep
- * equality. If they are deeply equal, returns the JS string.
- * @param {{}} data
- * @returns {string}
+ * Wraps and saves data to '../client/components/phone-input/data.js'
+ * @param {Object} data
  */
-function convertToJSStringAndVerify( data ) {
-	const dataString = toJSString( data );
-	const parsedData = parseJSStringToData( dataString );
-	assert.deepEqual( parsedData , data );
-	return dataString;
-}
-
-/**
- * Wraps and saves a given JS string to '../client/components/phone-input/data.js'
- * @param dataString
- */
-function saveToFile( dataString ) {
-	var scriptStr = '// Generated by build-metadata.js\n/* eslint-disable */\nmodule.exports = ' + dataString + ';\n/* eslint-enable */\n';
+function saveToFile( data ) {
+	var scriptStr = (
+		'/** @format */\n' +
+		'// Generated by build-metadata.js\n' +
+		'/* eslint-disable */\n' +
+		Object
+			.keys( data )
+			.map( key => `export const ${ key } = ${ JSON.stringify( data[ key ], null, 3 ) };\n` )
+			.join( '\n' ) +
+		'/* eslint-enable */\n'
```

Comment:
also need to add a default export unless you are sure this is never imported via default

Reply:
when I checked I found no instances of importing the default, only of importing `{ named, constants }` from it.